src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -41,18 +46,29 @@ export default function WebexRemoteMedia({className, meetingID}) {
         </AlertBanner>
       ) : (
         <>
-          {!hasMedia ? (
+          {hasMedia && hasMembers && (
```

Comment:
This is why I don't like the `&&` format... I personally find it harder to reason about when there are multiple conditions.
Maybe we can rearrange the conditions/JSX?
```
  const remoteDisplay = hasOtherMembers ? <media JSX> : <waiting for others to join JSX>;

...

  hasMedia ? remoteDisplay : <connecting JSX>
```

if that makes sense

Reply:
There were multiple nested ifs causing eslint issues, but I will re-investigate